CMakeLists.txt

```diff
@@ -217,6 +215,8 @@ if(NOT USE_SGX STREQUAL "OFF")
   add_dependencies(tvm_runtime sgx_edl tvm_t)
   install(TARGETS tvm_t ARCHIVE DESTINATION lib${LIB_SUFFIX})
 endif()
+=======
+>>>>>>> f531c1a7... Remove SGX in preparation for adding SGX
```

Comment:
Looks like a failed git merge

Reply:
No worries. I just started being "on vacation," so I don't have access to my usual gnu/linux toolchain to test. This is mainly just me pushing to my remote so that I can re-pull and test. Suffice to say that I will not remove the "Draft" modifier until the PR is truly ready :)